docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.4"
+
+x-app: &app_base
+  depends_on:
+    - db
+  image: ${IMAGE_NAME}
+  environment:
+    - DATABASE_URL=postgres://postgres:password@db
+    - RAILS_ENV
+    - RACK_ENV
+    - DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL
+    - AUTH_SECRET_TOKEN
+    - MAILER_SENDER_ADDRESS
+    - PASSWORD_RECOVERY_LINK_TEMPLATE
+  build:
+    context: .
+    args:
+      - BUNDLE_WITHOUT="${BUNDLE_WITHOUT}"
+      - BUNDLER_VERSION=2.1.4
+      - FOLDERS_TO_REMOVE=""
+  links:
+    - db
+  volumes:
+    - ruby-bundle:/usr/local/bundle
```

Comment:
Maybe also use separate volume for `node_modules` folder?  Seems like now container will use `node_modules` from the host machine? (because of `.:/app` volume). If so, I think it's no good because of different dependencies between diff machines (MacOS Host, Linux container)